assets/javascripts/discourse/templates/components/bread-crumbs-assigned.hbs

```diff
@@ -0,0 +1,19 @@
+{{#each categoryBreadcrumbs as |breadcrumb|}}
```

Comment:
Why do you need to create this new bread crumb component? Can't you re-use the [one in core](https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/app/templates/components/bread-crumbs.hbs)?

Reply:
as we need to add a `categoryId` variable in bread-crumbs-assigned to add in query-params so, I thought this will be the best approach. your thoughts?

Reply:
Do you need the `categoryId` when you can extract the `id` of the `category`?

Reply:
That's right but we need to override `onChange` in category-drop, so may I keep that empty? as in `category-drop`, `onChange` method redirects to URL of that category

Reply:
I think we need to do that, as we need to override category-drop and to override that too we need to override `bread-crumbs` template